.github/scripts/inventory-evaluate.py

```
@@ -0,0 +1,101 @@
+                break
+    if header_idx is None:
+        sys.exit(f"PARSE ERROR: no inventory table header in {path}")
+    rows = []
+    for line in lines[header_idx + 2:]:
+        if not line.lstrip().startswith("|"):
+            break
+        cells = split_row(line)
+        if len(cells) != len(COLUMNS):
+            # Tolerate trailing/leading empties from formatting; otherwise skip.
+            continue
+        rows.append(dict(zip(COLUMNS, cells)))
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)